packages/react-magma-dom/src/components/Tabs/TabsScrollSpyContainer.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useCallback } from 'react';
+import styled from '@emotion/styled';
+import { ScrollSpy } from './utils';
+import { Tabs, TabsOrientation } from './Tabs';
+import { Tab } from './Tab';
+import { TabsContainer } from './TabsContainer';
+import { toCamelCase } from '../../utils';
+
+export interface TabScrollSpyContainerProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  /*
+   * Adds an icon to the navigation Tab.
+   */
+  icon?: React.ReactElement<any> | React.ReactElement<any>[];
+  isInverse?: boolean;
+  /*
+   * For custom click events.
+   */
+  onClick?: () => void;
+  /**
+   * @internal
+   */
+  testId?: string;
+}
+
+const StyledTabs = styled(Tabs)`
+  flex: 0 auto;
+  position: sticky;
+  top: 0;
+  li {
+    &:after {
+      transition: none;
+    }
+  }
+`;
+
+const StyledContentWrapper = styled.div`
+  flex: 1;
+`;
+
+export const TabsScrollSpyContainer = React.forwardRef<
+  HTMLDivElement,
+  TabScrollSpyContainerProps
+>((props, ref) => {
+  const { children, isInverse, testId, ...other } = props;
+
+  const [options, setOptions] = React.useState([]);
+
+  const [isActive, setIsActive] = React.useState(0);
+
+  const [activeIndex, setActiveIndex] = React.useState(0);
+
+  //Window scroll override
+  React.useEffect(() => {
+    const html = document.querySelector('html');
+    html.style.scrollBehavior = 'auto';
+  }, []);
+
+  //Sets the active tab state on scroll
+  React.useEffect(() => {
+    options.map((option: any) => {
+      // const lastIndex = options.length - 1;
+
+      // const scrollBottom =
+      //   window.innerHeight + window.scrollY >=
+      //   document.documentElement.scrollHeight - 600;
+
+      // if (scrollBottom && activeIndex === lastIndex - 1) {
+      //   setActiveIndex(lastIndex);
+      //   console.log('bottom', lastIndex, typeof option.index);
+      // }
```

Comment:
Before we merge, let's figure out what's happening with this. If we want to keep it for later, let's add a `TODO` comment. If we don't think we'll get to this, it should be removed.